mne/coreg.py

```diff
@@ -1242,3 +1245,613 @@ def _scale_xfm(subject_to, xfm_fname, mri_name, subject_from, scale,
+    @property
+    def _changes(self):
+        move = np.linalg.norm(self._last_translation - self._translation) * 1e3
+        angle = np.rad2deg(_angle_between_quats(
+            rot_to_quat(rotation(*self._rotation)[:3, :3]),
+            rot_to_quat(rotation(*self._last_rotation)[:3, :3])))
+        percs = 100 * (self._scale - self._last_scale) / self._last_scale
+        return move, angle, percs
+
+    @property
+    def _transformed_dig_hpi(self):
+        return apply_trans(self._hsp_trans, self._dig.hpi_points)
+
+    @property
+    def _transformed_dig_eeg(self):
+        return apply_trans(self._hsp_trans, self._dig.eeg_points)
+
+    @property
+    def _transformed_dig_extra(self):
+        return apply_trans(self._hsp_trans, self._dig.extra_points)
+
+    @property
+    def _hsp_trans(self):
+        if self._coord_frame == 'head':
+            t = np.eye(4)
+        else:
+            t = self._head_mri_t
+        return t
+
+    @property
+    def _transformed_orig_dig_extra(self):
+        return apply_trans(self._hsp_trans, self._dig._extra_points)
+
+    @property
+    def _nearest_transformed_high_res_mri_idx_orig_hsp(self):
+        return self._nearest_calc.query(
+            apply_trans(self._head_mri_t, self._dig._extra_points))[1]
+
+    @property
+    def _nearest_transformed_high_res_mri_idx_hpi(self):
+        return self._nearest_calc.query(
+            apply_trans(self._head_mri_t, self._dig.hpi_points))[1]
+
+    @property
+    def _has_hpi_data(self):
+        return (self._has_mri_data and
+                len(self._nearest_transformed_high_res_mri_idx_hpi) > 0)
+
+    @property
+    def _nearest_transformed_high_res_mri_idx_eeg(self):
+        return self._nearest_calc.query(
+            apply_trans(self._head_mri_t, self._dig.eeg_points))[1]
+
+    @property
+    def _has_eeg_data(self):
+        return (self._has_mri_data and
+                len(self._nearest_transformed_high_res_mri_idx_eeg) > 0)
+
+    @property
+    def _nearest_transformed_high_res_mri_idx_rpa(self):
+        return self._nearest_calc.query(
+            apply_trans(self._head_mri_t, self._dig.rpa))[1]
+
+    @property
+    def _nearest_transformed_high_res_mri_idx_nasion(self):
+        return self._nearest_calc.query(
+            apply_trans(self._head_mri_t, self._dig.nasion))[1]
+
+    @property
+    def _nearest_transformed_high_res_mri_idx_lpa(self):
+        return self._nearest_calc.query(
+            apply_trans(self._head_mri_t, self._dig.lpa))[1]
+
+    @property
+    def _has_lpa_data(self):
+        return (np.any(self._mri.lpa) and np.any(self._dig.lpa))
+
+    @property
+    def _has_nasion_data(self):
+        return (np.any(self._mri.nasion) and np.any(self._dig.nasion))
+
+    @property
+    def _has_rpa_data(self):
+        return (np.any(self._mri.rpa) and np.any(self._dig.rpa))
+
+    def _nearest_calc_default(self):
+        return _DistanceQuery(
+            self._processed_high_res_mri_points * self._scale)
```

Comment:
No need for this function, all of the `_default` methods were just to make Traits happy. Just set this in the constructor